config/capz/kustomization.yaml

```diff
@@ -51,8 +52,7 @@ replacements:
       index: 1
     select:
       annotationSelector: cert-manager.io/inject-ca-from
-
-- source: # SERVICE_NAMESPACE
+- source:
```

Comment:
```suggestion
- source: # SERVICE_NAMESPACE
```
Some more comments to preserve.

Reply:
added via 7902bf3d8168dafa5054c010f64c089df370b3ce